src/ws/messageTypes.ts

```diff
@@ -50,38 +50,24 @@ export interface MessageHeaders {
   message_id: string;
 }
 
-export interface ReplyMessageHeaders extends MessageHeaders {
```

Comment:
This additional header field not needed. Now only one class in MessageHeader family, removing the need for `MessageHeaderMap`, etc.